src/router.js

```diff
@@ -9,6 +9,12 @@ angular.module('bicker_router', ['ngAnimate']).run(function (State, Route, $loca
   });
 
   $rootScope.$on('$locationChangeSuccess', function (e, newUrl) {
+
+    setTimeout(function() {
+      const eventData = {'viewName': Route.getCurrentViewName()};
+      $rootScope.$emit('bicker_router.$locationChangeSuccess', eventData);
+    }, 1)
```

Comment:
I don't like this setTimeout, is there a way around using it? it seems to send the previous viewName without it.